packages/server/package.json

```diff
@@ -13,6 +13,10 @@
     "envalid": "^8.0.0",
     "fastify": "^5.0.0",
     "fastify-type-provider-zod": "^4.0.2",
+    "jose": "^5.9.6",
+    "jsonwebtoken": "^9.0.2",
+    "node-fetch": "2",
```

Comment:
I'm not sure we need `node-fetch` with a recent Node version? The country-config package might need it (as @types/node-fetch) as it's exported into the country-configuration but otherwise we could be fine with the default one